clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -76,6 +75,8 @@
   private final Registry registry;
   private final Id metricId;
   private final Provider<AwsLookupUtil> awsLookupUtil;
+  private final long TIME_UPDATE_THRESHHOLD_MS = 5000;
+  private final long ITEMS_CHANGED_THRESHHOLD = 1000;
```

Comment:
Since these are going to be modified by a different thread, should we be using `AtomicLong` instead?

Reply:
they don't get modified

Reply:
Whoops, I meant to comment this on this part: https://github.com/spinnaker/clouddriver/pull/3224/files#diff-ba906249bad8b13af34d50731bc17b94R162